tests/session/session-isInTransaction-001.phpt

```diff
@@ -0,0 +1,74 @@
+--TEST--
+MongoDB\Driver\Session::isInTransaction()
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_crypto() ?>
+<?php skip_if_server_version('<', '4.0'); ?>
+<?php skip_if_not_replica_set(); ?>
+<?php skip_if_not_clean(DATABASE_NAME, COLLECTION_NAME); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(URI);
+
+/* Create collections as that can't be (automatically) done in a transaction */
+$cmd = new \MongoDB\Driver\Command([
+    'create' => COLLECTION_NAME,
+]);
+$manager->executeCommand(DATABASE_NAME, $cmd);
+
+/* Start a session */
+$session = $manager->startSession();
+
+/* Empty transaction, and aborted empty transaction */
+var_dump($session->isInTransaction());
+$session->startTransaction();
+var_dump($session->isInTransaction());
+$session->abortTransaction();
+var_dump($session->isInTransaction());
+
+/* Empty transaction, and committed empty transaction */
+var_dump($session->isInTransaction());
+$session->startTransaction();
+var_dump($session->isInTransaction());
+$session->commitTransaction();
+var_dump($session->isInTransaction());
```

Comment:
Given that transactions are started lazily during the first command, how does this interact when the only operation is an abort or commit? I assume the driver is still sending something in this case and the server just registers it as a no-op?

Perhaps a question for @ShaneHarvey.